### PR TITLE
New bookin request

### PR DESCRIPTION
**Pull request by chizukano**
Added action new/create to Bookings controller. After we click the submit, the link is still pending and nothing shows. I will come back here to redirect the link to bookings/:id(show).

localhost_3000_gadgets_31_bookings_new

### CODE REVIEW

**Review by David-Nicolson100 — APPROVED** [review]

File: app/controllers/bookings_controller.rb [hunk 1]
```diff
@@ -1,5 +1,26 @@
...
   def new
+    @gadget = Gadget.find(params[:gadget_id])
     @booking = Booking.new
   end
+
+  def create
+    @gadget = Gadget.find(params[:gadget_id])
+    @booking = Booking.new(booking_params)
+    @gadget.booking = @booking
+    @booking.saves
```
Comment (brianna-foley) [inline app/controllers/bookings_controller.rb:11]: change to @booking.save

File: app/controllers/bookings_controller.rb [hunk 2]
```diff
@@ -1,5 +1,26 @@
...
+
+    #For the future link for bookings/show page
+    # if @booking.saves
```
Comment (brianna-foley) [inline app/controllers/bookings_controller.rb:14]: change to @booking.save

**Review by brianna-foley — CHANGES_REQUESTED** [review]
This needs a fix and the save and redirect or render pattern

**Review by brianna-foley — APPROVED** [review]
Closes #31 
Good to go!

**Pull request merged**